src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/ExportLaunchProfileExtensionValueProviderAttribute.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Debug
+{
+    /// <summary>
+    /// Exports a <see cref="ILaunchProfileExtensionValueProvider" /> or <see cref="IGlobalSettingExtensionValueProvider"/>.
```

Comment:
nit: this isn't the class that exports those interfaces, right?

Reply:
If you want to export an `ILaunchProfileExtensionValueProvider` or an `IGlobalSettingExtensionValueProvider` you put this attribute on the type, yes.